docs/API.md

```diff
@@ -275,6 +276,58 @@ main = do
 
 ```
 
+<a name="listObjectsV1"></a>
+### listObjectsV1 :: Bucket -> Maybe Text -> Bool -> C.Producer Minio ObjectInfo
+
+List objects in the given bucket (version 1)
```

Comment:
We should note here that this is an older API and is provided for compatibility. Also, we should mention that `listObjects` implements the V2 API.

Reply:
Sure.